src/test/java/org/cactoos/map/NoNullsTest.java

```diff
@@ -251,8 +250,7 @@ public void putWithNullValue() {
         );
     }
 
-    @Test
-    @Ignore
+    @Test(expected = IllegalStateException.class)
```

Comment:
final V result = this.map.put(key, value);
        if (result == null) {
            throw new IllegalStateException(
                new UncheckedText(
                    new FormattedText(
                        "Value returned by #put(%s,%s) is NULL",
                        key, value
                    )
                ).asString()
            );
        }